lib/dependencies/HarmonyExportInitFragment.js

```diff
@@ -145,11 +145,12 @@ class HarmonyExportInitFragment extends InitFragment {
 				? `/* unused harmony export ${first(this.unusedExports)} */\n`
 				: "";
 		const definitions = [];
-		for (const [key, value] of this.exportMap) {
+		const orderedExportMap = Array.from(this.exportMap.keys()).sort();
+		for (const key of orderedExportMap) {
```

Comment:
```suggestion
		const orderedExportMap = Array.from(this.exportMap).sort(([a], [b]) => a < b ? -1 : 1);
		for (const [key, value] of orderedExportMap) {
```

That should be a little bit more performant as it isn't calling `get()` many times.